.github/workflows/ansible-lint.yml

```diff
@@ -0,0 +1,14 @@
+name: ansible-lint
+on: [pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+          architecture: x64
+      - run: pip install -r <(grep ansible tests/requirements.txt) ansible-lint==4.3.7
+      - run: ansible-lint -v --force-color ./*.yml
```

Comment:
want to add this to the tox.ini? makes it easier to anyone to run this locally via tox